### DISCUSSION TITLE
Error after updating from salvo 0.60.0 to 0.63.0

### DISCUSSION THREAD

**Discussion by josejachuf** [salvo-rs/salvo#565] [Q&A] [answered]
Hi, 
After updating from salvo 0.60.0 to 0.63.0 this error occurs   

> Compiling brotli-decompressor v2.3.4
> error[E0599]: no method named `bind_device` found for struct `Socket` in the current scope
>    --> /home/jose/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hyper-util-0.1.1/src/client/legacy/connect/h
> ttp.rs:704:14
>     |
> 703 | /         socket
> 704 | |             .bind_device(Some(interface.as_bytes()))
>     | |             -^^^^^^^^^^^ method not found in `Socket`
>     | |_____________|
>     | 
> 
> error[E0599]: no method named `with_retries` found for struct `TcpKeepalive` in the current scope
>    --> /home/jose/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hyper-util-0.1.1/src/client/legacy/connect/http.rs:130:12
>     |
> 130 |         ka.with_retries(retries)
>     |            ^^^^^^^^^^^^ help: there is a method with a similar name: `with_time`
> 
> For more information about this error, try `rustc --explain E0599`.
> error: could not compile `hyper-util` (lib) due to 2 previous errors
> warning: build failed, waiting for other jobs to finish...

Jose

**Comment by astoring** [comment, marked as answer]
cargo update
cargo clean

**Reply by josejachuf** [reply]
I did the following and did not work:
rustup update
cargo clean
cargo build

But with your indications it worked perfect

Thanks!!
